images/coredns/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM centos:7
+
+RUN yum install -y golang
+
+ENV GOPATH /go
+RUN mkdir $GOPATH
+
+COPY . $GOPATH/src/github.com/coredns/coredns
+
+RUN cd $GOPATH/src/github.com/coredns/coredns \
+    && go build -o coredns . \
+    && cp $GOPATH/src/github.com/coredns/coredns/coredns /usr/bin/
+
+RUN yum remove -y golang
+
+ENTRYPOINT ["/usr/bin/coredns"]
+
+USER 1001
+
+LABEL io.k8s.display-name="CoreDNS" \
+      io.k8s.description="CoreDNS is a DNS server that chains plugins" \
+      maintainer="coredns-discuss@googlegroups.com"
```

Comment:
Us

Reply:
Used aos-devel@redhat.com, is there a better address?

Reply:
dev@lists.openshift.redhat.com would make more sense since it's public.  The [openshift-ansible Dockerfile uses](https://github.com/openshift/openshift-ansible/blob/95c564a82d1efa998ff0f95187fbb701f3a7622c/images/installer/Dockerfile#L3) this address.

Reply:
Fixed